e/src/main/java/iam/thevoid/e/OptionalExtensions.kt

```diff
@@ -4,75 +4,56 @@ package iam.thevoid.e
  * SAFE
  */
 
```

Comment:
At least old realization must be marked as deprecated with warning, but not completely removed

Also, as if 'safe' is function now, maybe there is could be default value in function arguments? Looks like 'elvis' but why not? With default value we provide single interface to safety